client/config.yaml

```diff
@@ -14,15 +14,12 @@
 
 gcp_settings:
   # GCP project number.
-  project_number: 'XXXXXXXXXX'
+  project_number: 'translation-test-external'
```

Comment:
I'm a bit surprised, `project_number` would make me thing only numeric IDs are allowed here

Reply:
I was actually thrown off by this as well.